getting-started/authenticating-users/app.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2019 Google LLC
```

Comment:
Add tests if possible/relevant

Reply:
TBD. The only thing testable is validateAssertion, and that would be pointless since the test would have to generate the assertion and then validate it.